packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -736,6 +736,7 @@ export interface DataGridPropsWithoutDefaultValue<R extends GridValidRowModel =
   getRowId?: GridRowIdGetter<R>;
   /**
    * If `true`, a loading overlay is displayed.
+   * @default false
```

Comment:
I just had to update this row and `pnpm proptypes && pnpm docs:api` did the rest.

Reply:
We don't usually add `@default` to the values in `DataGridPropsWithoutDefaultValue`. Should we also move the `loading` prop to `DataGridPropsWithDefaultValues`?

Reply:
No preferences, it adds a bit of bundle size but is also clearer.